### ISSUE TITLE
Set up Weights & Biases

### ISSUE THREAD

**Issue by luciaquirke** [AISC-understanding-search/maze-transformer#90]
We want a new w&b dashboard to help track experiments and share results.

The MVP for this should move all logging statements to log to a w&b dashboard (?). The next step would be to record evaluations in the dashboard. 

We plan not to prioritise this until we have a working model. We may want to split this into multiple tickets before starting to work on it.
